spec/factories/document_type_factory.rb

```diff
@@ -9,7 +9,13 @@
     contents { [] }
     tags { [] }
     guidance { [] }
-    publishing_metadata { DocumentType::PublishingMetadata.new }
+    publishing_metadata {
+      DocumentType::PublishingMetadata.new(
+        "schema_name": "news_article",
+        "rendering_app": "government-frontend",
```

Comment:
It looks like these are really specific values towards news, whereas the rest of this factory is rather generic. 

It'd be more fitting with the existing patterns of this factory if these were randomly generated.

Though I actually think you can avoid making a change here - I'll reference that in my next comment.

Reply:
Sure, IIve removed the bespoke behaviour and created the specific document type i needed in the test itself